src/components/download/__tests__/DownloadLegend.spec.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { DownloadLegend } from '../DownloadLegend';
+
+describe('DownloadLegend', () => {
+    const classes = {
+        root: 'root-class',
+        topleft: 'topleft-class',
+        topright: 'topright-class',
+        bottomleft: 'bottomleft-class',
+        bottomright: 'bottomright-class',
+        name: 'name-class',
+    };
+
+    const layers = [
+        {
+            id: 'layer-1',
+            legend: {
+                title: 'Layer 1',
+                period: '2018',
+            },
+        },
+        {
+            id: 'layer-2',
+        },
+        {
+            id: 'layer-3',
+            legend: {
+                title: 'Layer 2',
+                period: '2017',
+            },
+        },
+    ];
+
+    const renderComponent = props =>
+        shallow(
+            <DownloadLegend
+                classes={classes}
+                position="bottomright"
+                layers={[]}
+                showName={false}
+                {...props}
+            />
+        );
+
+    it('Should render a legend component', () => {
+        expect(renderComponent().exists()).toBe(true);
+    });
+
+    it('Should set legend position class', () => {
+        const wrapper = renderComponent();
+        expect(wrapper.hasClass('bottomright-class')).toBe(true);
```

Comment:
Minor: these should probably reference `classes.bottomright` etc. instead of hard-coding the class name.

Reply:
Same with this whole file

Reply:
Fixed